### PR TITLE
fix hooks and actions

### PR DESCRIPTION
**Pull request by rileylnapier**
## Description

- get unread message count payload type was wrong
- don't fetch "initial state" if we are only using hooks

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> `Fix [#1]()`

### CODE REVIEW

File: packages/react-hooks/src/inbox/elemental-inbox/use-inbox-actions.ts [hunk 1]
```diff
@@ -77,7 +77,6 @@ const useElementalInboxActions = (): IInboxActions => {
         type: "inbox/FETCH_UNREAD_MESSAGE_COUNT",
         payload: () =>
           inboxClient.getInboxCount({
-            tags: [],
```
Comment (rileylnapier) [inline packages/react-hooks/src/inbox/elemental-inbox/use-inbox-actions.ts:80]: this was pulling back 0 for count

File: packages/react-hooks/src/inbox/use-inbox-actions.ts [hunk 2]
```diff
@@ -56,11 +66,29 @@ const useInboxActions = (): IInboxActions => {
...
   const events = Events({ client: courierClient });
   const messages = Messages({ client: courierClient });
+  const initialState = InitialState({ client: courierClient });
 
   return {
-    init: (payload) => {
+    init: async (payload) => {
       dispatch(initInbox(payload));
 
+      const response = await initialState.getInitialState({
```
Comment (rileylnapier) [inline packages/react-hooks/src/inbox/use-inbox-actions.ts:75]: if someone was only to use the hooks... i feel like we maybe shouldn't fetch initial state right?  we should only do this initial state stuff if we are using the components?

Comment (rileylnapier) [reply packages/react-hooks/src/inbox/use-inbox-actions.ts:75]: jk this "init" action only gets fired if you are using the components so it should be fine

File: packages/storybook/stories/inbox/full-page-inbox.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+          justifyContent: "space-between",
+        }}
+      >
+        <div>
+          <ReactMarkdown>{`## Example`}</ReactMarkdown>
+          <ReactMarkdown>{`\`\`\`javascript\n${fullPageInboxHooksString}\n\`\`\``}</ReactMarkdown>
+        </div>
+        <CourierProvider
+          wsOptions={{
+            url: "wss://icnrz8ttcf.execute-api.us-east-1.amazonaws.com/staging",
```
Comment (rileylnapier) [inline packages/storybook/stories/inbox/full-page-inbox.stories.tsx:30]: not sure if this matters i left staging on here.  it was how i tested the CPP changes

**Review by drew-y — APPROVED** [review]

**Review by drew-y — DISMISSED** [review]

**Review by drew-y — APPROVED** [review]

**Review by drew-y — APPROVED** [review]

**Pull request merged**